django/contrib/auth/hashers.py

```diff
@@ -396,7 +397,7 @@ def encode(self, password, salt):
         argon2 = self._load_library()
         params = self.params()
         data = argon2.low_level.hash_secret(
-            password.encode(),
+            force_bytes(password),
             salt.encode(),
```

Comment:
In the previous PR, the salt(s) had `force_bytes(salt)` etc 
Can you explain what motivated this update?

Reply:
I added support for accepting `str` and `bytes` types for both password and salt.